source/openapi-admin-v3.yaml

```diff
@@ -5063,15 +5062,24 @@ components:
                   type: boolean
                   default: false
                   description: |-
-                    If `true`, enabling the trigger after it was disabled
-                    will not invoke events that occurred while the trigger
+                    If `true`, enabling the Trigger after it was disabled
+                    will not invoke events that occurred while the Trigger
                     was disabled.
+                tolerate_resume_errors:
+                  type: boolean
+                  default: false
+                  description: |-
+                    If `true`, when this Trigger falls off the oplog
```

Comment:
I am not sure if we use this wording elsewhere, but if we do then I am fine with leaving it. The Trigger doesn't fall off the oplog, it is the **resume token** that falls off the oplog (and cannot be found). The rest of this section looks good.

Reply:
+1 for rewording

Reply:
sure. open to it. this was the exact phrasing taken from the App Services UI though. 

the UI says:

> If enabled, the Trigger will automatically resume processing from the most recent event onwards in the event that this Trigger falls off the oplog or the resume token cannot be found.

we should consider a change there as well.

Reply:
Nice. I can update on our end. Do you have phrasing you want to use that I can align to or should I grab my design team to come up with something?

Reply:
hmm maybe smthn like (very similar to the docs): 

> If enabled, when this Trigger's resume token cannot be found in the cluster's oplog, the Trigger automatically resumes processing events at the next relevant change stream event.

Reply:
Sounds good to me. I will start working on propagating those changes and have our design team update our side